subsys/nrf_security/src/drivers/cracen/sxsymcrypt/src/blkcipher.c

```diff
@@ -381,7 +381,9 @@ int sx_blkcipher_status(struct sxblkcipher *c)
 	}
 
 #if CONFIG_DCACHE
-	sys_cache_data_invd_range((void *)&c->extramem, sizeof(c->extramem));
```

Comment:
invalidate range should be avoided as it could lead to memory loss on that cacheline if something else on the line was written by the core, e.g. if extramem is somewhere on the stack